### ISSUE TITLE
Sync Issue?[BUG]

### ISSUE THREAD

**Issue by VladZeInhaler** [Glagan/SyncDex#4]
Hello, 
I've was recently using mal-sync and MyMangadex in combination to save my list on mal and anilist but I downloaded this extension since I got a notification about it. Today, I downloaded the extension for edge to see if it worked and it did for the most part but the stuff that I am reading wasn't automatically syncing so I go to manually sync it and it just gets stuck. At the moment, it was been stuck for 10 minutes. After that, I thought it was an issue with edge to I downloaded chrome and tried it but it is also stuck when I try to manually sync it.  Sync also never loaded for me. I don't know if this is a bug or I am just an idiot and missed something. Thanks for the help either way. 
**On Edge:**
https://gyazo.com/32bf08c58015339c003e4c3d4f1b6124
https://gyazo.com/77e8383d4480fa63a7c12f778acf4a0e
**On Chrome:**
https://gyazo.com/a51204ec3e8b535505b2d9d03de1d1d6
**Link to Manga I was reading:**
https://mangadex.org/title/35871/a-rank-boukensha-no-slow-life

**Comment by Glagan** [comment]
This is most likely a bug, could you post your logs ? You can copy/paste them with the "Debug" button on the left in the options.

**Comment by VladZeInhaler** [comment]
```
{"lastSync":1608963222652,"logs":[{"d":1608957384271,"msg":"Installation version 0.1.4"},{"d":1608961961314,"msg":"Uploaded Local Save to Google Drive"},{"d":1608961961316,"msg":"Synced your save with GoogleDrive"},{"d":1608962017789,"msg":"Importing lists"},{"d":1608962017790,"msg":"Importing AnimePlanet"},{"d":1608962022688,"msg":"Imported AnimePlanet in 4898ms"},{"d":1608962022689,"msg":"Importing Kitsu"},{"d":1608962027937,"msg":"Imported Kitsu in 5248ms"},{"d":1608962027938,"msg":"Importing MangaUpdates"},{"d":1608962035445,"msg":"Imported MangaUpdates in 7507ms"},{"d":1608962035446,"msg":"Importing MyAnimeList"},{"d":1608962044303,"msg":"Uploaded Local Save to Google Drive"},{"d":1608962044306,"msg":"Synced your save with GoogleDrive"},{"d":1608962045617,"msg":"Imported MyAnimeList in 10171ms"},{"d":1608962045618,"msg":"Importing Anilist"},{"d":1608962059192,"msg":"Imported Anilist in 13574ms"},{"d":1608962059195,"msg":"Done Importing lists"},{"d":1608962107812,"msg":"Uploaded Local Save to Google Drive"},{"d":1608962107814,"msg":"Synced your save with GoogleDrive"},{"d":1608962594633,"msg":"Uploaded Local Save to Google Drive"},{"d":1608962594636,"msg":"Synced your save with GoogleDrive"},{"d":1608962769808,"msg":"Uploaded Local Save to Google Drive"},{"d":1608962769810,"msg":"Synced your save with GoogleDrive"},{"d":1608963222670,"msg":"Uploaded Local Save to Google Drive"},{"d":1608963222672,"msg":"Synced your save with GoogleDrive"},{"d":1608963367731,"msg":"[object Object]\nStack: undefined"}],"options":{"acceptLowScore":false,"autoSync":true,"biggerHistory":true,"chapterStatus":false,"chaptersSaved":400,"checkOnStartup":true,"checkOnStartupCooldown":10,"checkOnStartupMainOnly":true,"colors":{"higherChapter":"transparent","highlights":["rgba(82, 190, 90, 0.6)","rgba(107, 177, 95, 0.6)","rgba(55, 168, 61, 0.6)"],"lowerChapter":"rgba(180, 102, 75, 0.5)","nextChapter":"rgba(104, 115, 251, 0.4)","openedChapter":"rgba(28, 135, 141, 0.4)"},"confirmChapter":true,"errorNotifications":true,"favoriteLanguage":"all","groupTitlesInLists":true,"hideHigher":false,"hideLast":false,"hideLower":true,"highlight":true,"linkToServices":true,"mainService":"al","mdUpdateSyncDex":true,"noReloadStatus":true,"notifications":true,"originalThumbnail":false,"overviewMainOnly":true,"progressInThumbnail":true,"saveOnlyHigher":true,"saveOnlyNext":false,"saveOpenedChapters":true,"separateLanguages":true,"services":["al","mal","mu","ku","ap"],"subVersion":4,"thumbnail":true,"thumbnailMaxHeight":80,"tokens":{"anilistToken":"set","kitsuToken":"set","kitsuUser":"set"},"updateMD":true,"updateOnlyInList":false,"useMochi":true,"version":0.1},"saveSync":{"expires":1608965556825,"id":"1jnMqD-3mYWifwAUSiBnaA-fFQ9c5EJVmv0GfLN0zvrzzdYRUSw","refresh":"set","service":"GoogleDrive","token":"set"}}
```

**Comment by Glagan** [comment]
I can't see the error in your logs, and it's weird that it's just hanging, if there was an error it would just fail and display a message.  
I added some code to try to catch the error if it happen again, and I can only just wait since I can't reproduce the error.  
Is your network stablen, maybe there is some disconnects and I'm not handling correctly.

Edit: If you keep having this issue in version ``0.1.5`` please post your logs.

**Comment by Glagan** [comment]
Fixed in version ``0.1.6`` if you still have **AnimePlanet** enabled.
